s4e-backend/src/test/java/pl/cyfronet/s4e/controller/InstitutionControllerTest.java

```diff
@@ -29,6 +29,7 @@
 @AutoConfigureMockMvc
```

Comment:
In previous PR I've created generic use helper (factory) to generate unique element mock to create
- institution
- group
- user
- invitation

it's called `InvitationHelper`. It would be nice if you replace all mocks in this test and recall class with a more generic name like `Factory`.

Even if it isn't ticket scope for such changes
- code will be simpler
- there will be fewer lines of code (around 50 - 70)
- it will be a good moment to propagate factory methodology from now on
- you will be not worried about fields and names any longer
- if there will be a change in a model (DB), mock will be changed in one place (factory)

(optional)
You can generate all necessary mocks in before function